examples/src/main/java/tachyon/examples/JournalCrashTest.java

```diff
@@ -0,0 +1,377 @@
+     * @throws IOException
+     */
+    public void writeStatus(String localFile) throws IOException {
+      ObjectOutputStream obOut = new ObjectOutputStream(new FileOutputStream(localFile));
+      obOut.writeObject(mOpTypes);
+      obOut.writeObject(mSuccessNums);
+      obOut.writeObject(mWorkDirs);
+      obOut.close();
+    }
+  }
+
+  // The two Exit Codes are used to tell script if the test runs well.
+  private static final int EXIT_FAILED = 99;
+  private static final int EXIT_SUCCESS = 100;
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+
+  private static int sClientNum = 10;
+  private static TachyonURI sMasterAddress = null;
+  private static String sTestDir = null;
+  private static List<Thread> sClientThreadList = null;
+  private static List<ClientThread> sClientThreads = null;
+  private static String sTmpOutFile = null;
+
+  private static boolean checkStatus() throws Exception {
+    // Load the status.
+    StatusRecord status = StatusRecord.readStatus(sTmpOutFile);
+    List<ClientOpType> opTypes = status.getOpTypes();
+    List<Integer> successNums = status.getSuccessNums();
+    List<String> workDirs = status.getWorkDirs();
+
+    // Launch a Tachyon Client and connect to Master. Check if all the test operations are
+    // reproduced by Master successfully.
+    TachyonFS tfs = TachyonFS.get(sMasterAddress, new TachyonConf());
+    for (int clientIndex = 0; clientIndex < opTypes.size(); clientIndex ++) {
+      ClientOpType opType = opTypes.get(clientIndex);
+      String workDir = workDirs.get(clientIndex);
+      for (int s = 0; s < successNums.get(clientIndex); s ++) {
+        if (ClientOpType.CREATE_FILE == opType) {
+          if (tfs.getFileId(new TachyonURI(workDir + s)) == -1) {
+            tfs.close();
+            return false;
+          }
+        } else if (ClientOpType.CREATE_DELETE_FILE == opType) {
+          if (tfs.getFileId(new TachyonURI(workDir + s)) != -1) {
+            tfs.close();
+            return false;
+          }
+        } else if (ClientOpType.CREATE_RENAME_FILE == opType) {
+          if (tfs.getFileId(new TachyonURI(workDir + s + "-rename")) == -1) {
+            tfs.close();
+            return false;
+          }
+        }
+        //else if (ClientOpType.CREATE_TABLE == opType) {
+        //  if (tfs.getRawTable(new TachyonURI(workDir + s)).getId() == -1) {
+        //    tfs.close();
+        //    return false;
+        //  }
+        //}
+      }
+    }
+    tfs.close();
+    return true;
+  }
+
+  public static void main(String[] args) {
+    if (args.length < 4) {
+      System.out.println("java -cp tachyon-" + Version.VERSION + "-jar-with-dependencies.jar "
+          + "tachyon.examples.JournalCrashTest " + "<0:Run Clients|1: Check Status> "
+          + "<TachyonMasterAddress> <TestTachyonDir> <TmpOutFile> [-options]");
+      System.exit(EXIT_FAILED);
+    }
+
+    int runType = Integer.parseInt(args[0]);
```

Comment:
Why are there two different modes? Why doesn't the test run the clients, and then check the status at the end?